### PR TITLE
Fat 2404/e titles spec greening

### PR DESCRIPTION
**Pull request by kimkate303**

### CODE REVIEW

File: cypress/integration/eholdings/eholdings-titles.spec.js [hunk 1]
```diff
@@ -8,126 +6,102 @@ import eHoldingsPackages from '../../support/fragments/eholdings/eHoldingsPackag
...
-      eHoldingsTitles.openTitle();
-      eHoldingsTitle.waitPackagesLoading();
-      eHoldingsTitle.filterPackages(eHoldingsPackage.filterStatuses.selected, selectedResource.package);
-      eHoldingsTitle.waitPackagesLoading();
-      eHoldingsTitle.openResource();
-      eHoldingsResourceView.goToEdit();
-      eHoldingsResourceEdit.swicthToCustomCoverageDates();
+  it('C16994 Add a title in a package to holdings (spitfire)', { tags:  [testTypes.smoke, features.eHoldings, devTeams.spitfire] }, () => {
+    eHoldingsTitlesSearch.bySubject(testData.subjectTitle);
+    // findItem();
```
Comment (MaksimTerekhov) [inline cypress/integration/eholdings/eholdings-titles.spec.js:44]: remove comment pls

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/integration/eholdings/eholdings-titles.spec.js (hunk 1 above)
Comment (kimkate303) [reply cypress/integration/eholdings/eholdings-titles.spec.js]: done

File: cypress.json [hunk 2]
```diff
@@ -1,15 +1,16 @@
+
 {
-  "baseUrl": "https://cypress.ci.folio.org",
+  "baseUrl": "https://bugfest-mg-aqa.int.aws.folio.org/",
```
Comment (MaksimTerekhov) [inline cypress.json:3]: we don't need this changes :)

File: cypress/integration/eholdings/eholdings-packages.spec.js [hunk 3]
```diff
@@ -1,105 +1,82 @@
...
 import testType from '../../support/dictionary/testTypes';
 import TopMenu from '../../support/fragments/topMenu';
 import eHoldingsPackages from '../../support/fragments/eholdings/eHoldingsPackages';
 import eHoldingSearch from '../../support/fragments/eholdings/eHoldingsSearch';
 import eHoldingsPackagesSearch from '../../support/fragments/eholdings/eHoldingsPackagesSearch';
 import eHoldingsPackage from '../../support/fragments/eholdings/eHoldingsPackage';
-import permissions from '../../support/dictionary/permissions';
 import features from '../../support/dictionary/features';
-import users from '../../support/fragments/users/users';
+import devTeams from '../../support/dictionary/devTeams';
```
Comment (TetianaParanich) [inline cypress/integration/eholdings/eholdings-packages.spec.js:8]: please use uppercase for imports

Comment (kimkate303) [reply cypress/integration/eholdings/eholdings-packages.spec.js]: done

**Pull request merged**